tests/tests/test_integration/test_gui/test_tracks.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+brainreg_dir = Path.cwd() / "tests" / "data" / "brainreg_output"
+validate_tracks_dir = (
+    brainreg_dir / "manual_segmentation" / "standard_space" / "tracks"
+)
+
+
+@pytest.fixture
+def test_tracks_dir(tmpdir):
+    tmp_input_dir = tmpdir / "brainreg_output"
+    test_tracks_dir = (
+        tmp_input_dir / "manual_segmentation" / "standard_space" / "tracks"
+    )
+    return test_tracks_dir
+
+
+def test_track_widget_layer_numbers(
+    segmentation_widget_with_data_atlas_space, test_tracks_dir
+):
+    assert len(segmentation_widget_with_data_atlas_space.viewer.layers) == 4
+    assert len(segmentation_widget_with_data_atlas_space.track_layers) == 1
+
+
+def test_add_new_track(
+    segmentation_widget_with_data_atlas_space, test_tracks_dir
+):
+    segmentation_widget_with_data_atlas_space.track_seg.add_track()
+    assert len(segmentation_widget_with_data_atlas_space.viewer.layers) == 5
+    assert len(segmentation_widget_with_data_atlas_space.track_layers) == 2
+    assert (
+        segmentation_widget_with_data_atlas_space.track_layers[0].name
+        == "test_track"
+    )
+    assert (
+        segmentation_widget_with_data_atlas_space.track_layers[1].name
+        == "track_1"
+    )
+    assert (
+        len(segmentation_widget_with_data_atlas_space.track_layers[0].data)
+        == 6
+    )
+
+
+def test_add_existing_track(
+    segmentation_widget_with_data_atlas_space, test_tracks_dir
+):
+    segmentation_widget = segmentation_widget_with_data_atlas_space
+
+    # importing existing track
+    points = np.array([[0, 0, 0], [1, 1, 1], [2, 2, 2], [3, 3, 3]])
+    test_layer = segmentation_widget.viewer.add_points(
+        points, name="test_track2"
+    )
+    segmentation_widget.viewer.layers.selection.select_only(test_layer)
+    segmentation_widget.track_seg.add_track_from_existing_layer(overrride=True)
```

Comment:
Worth fixing this typo here and in the source code while we're in the area (not worth the overhead of opening an issue, but we'll forget otherwise)?
```suggestion
    segmentation_widget.track_seg.add_track_from_existing_layer(override=True)
```